bin/runtime-common/src/messages.rs

```diff
@@ -787,8 +795,12 @@ mod tests {
 
 		fn bridged_balance_to_this_balance(
 			bridged_balance: BridgedChainBalance,
+			bridged_to_this_conversion_rate_override: Option<FixedU128>,
 		) -> ThisChainBalance {
-			ThisChainBalance(bridged_balance.0 * BRIDGED_CHAIN_TO_THIS_CHAIN_BALANCE_RATE as u32)
+			let conversion_rate = bridged_to_this_conversion_rate_override
+				.map(|r| r.to_float() as u32)
+				.unwrap_or(BRIDGED_CHAIN_TO_THIS_CHAIN_BALANCE_RATE);
+			ThisChainBalance(bridged_balance.0 * conversion_rate)
```

Comment:
```suggestion
			ThisChainBalance(bridged_balance.0.saturating_mul(conversion_rate))
```

Reply:
It's just a test, so it should be fine :)

Reply:
Oh, sorry! didn't notice it's a test code.